docs/Linguistic Features in Turkish Word Representations.md

```diff
@@ -0,0 +1,89 @@
+# Linguistic Features in Turkish Word Representations
+
+
+
+The dataset is a collection of analogical reasoning task pairs prepared by O. Gungor, and E. Yildiz. It contains 4 sets of pairs with different types of affixes, and word vectors obtained by skip-gram algorithm in their studies. [GitHub Repository](https://github.com/onurgu/linguistic-features-in-turkish-word-representations/releases) of the dataset and [the paper](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7960223&tag=1) can be found by given links.
+
+## Dataset Details
+
+
+
+
+The pairs contains two words with their simple versions followed by affixed states. This dataset contains 4 subsets obtained by different pairings according to their affixes.
+
+| File name | Description | Number of pairs| 
+|-------|-------------|--|
+| noun_inflection_quads.txt     | Pairs of nouns with  inflections   |15133
+| verb_inflection_quads.txt    | Pairs of verbs with inflections     |14231|
+| derivation_quads.txt    | Pairs of words with derivational affixes    |4836|
+| both_inflection_derivation.quads.txt| Pairs of words with both inflectional and derivational affixes| 303
+
+
+### Samples
+
+Sample pairs that take inflection:
+
+
+```
+konu konuları yer yerleri
+konu konuları gün günleri
+konu konuları ülke ülkeleri
+konu konuları devlet devletleri
+konu konuları iş işleri
+konu konuları hak hakları
+```
+
+
+
+
+
+## Dataset Creation
+
+### Curation Rationale
+
+The dataset is motivated by the desire to advance the word vectors representations in Turkish language to be used in many areas of NLP. 
```

Comment:
The dataset is motivated by the desire to advance the word vector representations in the Turkish language to be used in many areas of NLP.